Emby.Dlna/Didl/StringWriterWithEncoding.cs

```diff
@@ -20,7 +21,7 @@ public StringWriterWithEncoding(IFormatProvider formatProvider)
         }
 
         public StringWriterWithEncoding(StringBuilder sb)
-            : base(sb)
+            : base(sb, CultureInfo.InvariantCulture)
```

Comment:
You should keep it as it was and disable this warning for this file to keep behavior consistent (and we should probably remove this file in the future)